deployment.policy.json

```diff
@@ -1,21 +1,29 @@
 {
-  "label": "web-hello-python Deployment Policy",
-  "description": "A Horizon Deployment Policy example to run web-hello-python",
-  "service": {
-    "name": "web-hello-python",
-    "org": "examples",
-    "arch": "*",
-    "serviceVersions": [
-      {
-        "version": "1.0.0",
-        "priority":{}
+    "label": "$SERVICE_NAME Deployment Policy",      
+    "description": "Policy to auto deploy $SERVICE_NAME",
+    "service": {       
+      "name": "$SERVICE_NAME",
+      "org": "$HZN_ORG_ID",    
+      "arch": "*",    
+      "serviceVersions": [ 
+        {
+          "version": "$SERVICE_VERSION",
+          "priority":{}
+        }
+      ]
+    },
+    "properties": [
+    ],
+    "constraints": [ 
```

Comment:
Both of these constraints being present in the service and deployment policy is somewhat redundant ... I think this is a good place to stress the different concerns the service policy and deployment policy are addressing and the different roles that might be creating each. I think of the deployment policy as the role of the business decision maker. This person from a high level knows where and what nodes they want the service deployed to, and they might not know or even care about the specifics of the service, like what language it's written in. For that reason I think we should **only** have the `"purpose == learning"` constraint in this file